packages/ag-charts-types/src/chart/chartBuilderOptions.ts

```diff
@@ -51,6 +51,8 @@ export type AgBaseFinancialPresetOptions = AgBasePresetOptions & Pick<AgCartesia
 
 export type AgFinancialChartOptions = AgFinancialChartPresets & AgBaseFinancialPresetOptions;
 
+export type AgGaugeChartOptions2 = AgBasePresetOptions & AgGaugeSeriesOptions;
```

Comment:
`AgBasePresetOptions` includes `data`, which probably isn't relevant given we have `value` too?